Dolapo47/Auto-Mart
# create route to retrieve all car ads on the app[finish #166819400]

#### What does this PR do?
- A summary of what this task is about
This Pr gives the admin access to all the cars on the app
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-verify if the user is an admin or not
-if the user is an admin he has access to the route
-import the necessary dependencies
-write the controller 
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
-navigate to the get all cars route
-send the request
-the admin retrieves all ads on the app
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166819400](https://www.pivotaltracker.com/n/projects/2346179)`)
